flybywiresim/simbridge
# feat(a380x): TERR on ND for A380X

- introduce TERR ON ND for the A380
- Changes extent of terrain map on a380x: Map also visible behind aircraft
- Remove clip mask / circular cut-off for a380x
- Fix peaks mode information in ROSE mode (for both a32nx and a380x)

**Testing instructions (A32NX and A380X):**
1. Download release candidate version from https://www.swisstransfer.com/d/37d8fdf4-89f0-4a3f-9f78-9732b8801543
2. Replace existing SimBridge installation with downloaded version, start SimBridge
3. Load into airport with known terrain and match navaids/waypoints/airports on the ND with elevation maps (e.g. the VFR layer on the Navigraph app)
4. Make sure that display and positions remain accurate in both ROSE and ARC modes


Example:
![grafik](https://github.com/flybywiresim/simbridge/assets/64070348/a144d382-10bb-4255-9860-4a3ede6a7809)
![grafik](https://github.com/flybywiresim/simbridge/assets/64070348/36426ed1-8287-4a4a-84fd-a1e297b3b05e)
